weblogic-batch/scripts/alert_functions

```diff
@@ -0,0 +1,67 @@
+#######################################################
+#
+# This file contains only functions for BMC Patrol
+# and email alerts
+#
+#######################################################
+
+
+## The function patrol_log_alert_chaps_f is used to write a particular string to a remote
+## log file so that BMC Patrol can immediately alert the Oncall officer.
+##
+## If used, the CHAPLOG variable is set by UNIX_OPS in all servers .profile and loaded by at user level login.
+##
+## The function accepts an enclosed string that MUST contain:
+## patrol_log_alert_chaps_f " [path] [script_name] [helpful_text] "
+##
+## The path is used to distinguish test and live systems so one script can be used throughout
+## helpful_text must show the on-call officer what failed and where to look
+##
+## Example:
+##   patrol_log_alert_chaps_f " `pwd`/`basename $0`: LetterProducer may be already running or connection error."
+##
+
+patrol_log_alert_chaps_f ()
+{
+  if [[ ${LIVE_ENVIRONMENT} == "true" ]]; then
+    echo `date`: alert_chaps \> $1  >> /apps/oracle/chaplive/log/chapspatrol.log
+    echo -e "To:CSI@companieshouse.gov.uk\nFrom:CSI@companieshouse.gov.uk\nSubject:CHAPS Alert: $1 \n$1" | msmtp -t
```

Comment:
The csi emails address is in here 7 times.  I know this is what is in the existing script, but maybe it is worth having an external var for that, and one for the service now email address too?  If you do that though, will probably need a function to check that the vars are defined and call that from each alert function?